<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Collocations Collector</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QFrame" name="frame">
        <property name="minimumSize">
         <size>
          <width>289</width>
          <height>186</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QSplitter" name="splitter">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="pushButtonOpen">
              <property name="text">
               <string>Open</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../../../../usr/share/icons/gnome/32x32/actions/document-open.png</normaloff>../../../../../usr/share/icons/gnome/32x32/actions/document-open.png</iconset>
              </property>
             </widget>
             <widget class="QPushButton" name="pushButtonSave">
              <property name="text">
               <string>Save</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../../../../usr/share/icons/gnome/32x32/actions/document-save.png</normaloff>../../../../../usr/share/icons/gnome/32x32/actions/document-save.png</iconset>
              </property>
             </widget>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QSplitter" name="splitter_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <widget class="QPushButton" name="pushButtonListen">
              <property name="text">
               <string>Listen to Selected Text</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>../../../../../usr/share/icons/gnome/32x32/actions/gtk-media-play-ltr.png</normaloff>../../../../../usr/share/icons/gnome/32x32/actions/gtk-media-play-ltr.png</iconset>
              </property>
             </widget>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTextEdit" name="textEditCollocationsList"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_2">
      <property name="minimumSize">
       <size>
        <width>196</width>
        <height>188</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QComboBox" name="comboBoxCollocations">
         <property name="editable">
          <bool>true</bool>
         </property>
         <property name="insertPolicy">
          <enum>QComboBox::InsertAtTop</enum>
         </property>
         <item>
          <property name="text">
           <string>due to the fact </string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>with respect to</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>with a view to</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>in accordance with</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pushButtonCollect">
         <property name="text">
          <string>Collect Examples ...</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../../../usr/share/icons/gnome/32x32/actions/find.png</normaloff>../../../../../usr/share/icons/gnome/32x32/actions/find.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QWebView" name="webViewWordList" native="true">
         <property name="url" stdset="0">
          <url>
           <string>about:blank</string>
          </url>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>692</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
   <slots>
    <slot>playPause()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOpen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>openFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonSave</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>saveExamples()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>468</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonCollect</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>collectExamples()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>109</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuBar</sender>
   <signal>hovered(QAction*)</signal>
   <receiver>MainWindow</receiver>
   <slot>addFiles()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>55</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonListen</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>listenToSelectedText()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>480</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>546</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openFile()</slot>
  <slot>saveExamples()</slot>
  <slot>listenExamples()</slot>
  <slot>collectExamples()</slot>
  <slot>playNext()</slot>
  <slot>addFiles()</slot>
  <slot>listenToSelectedText()</slot>
 </slots>
</ui>
